docs/content/guide/di.ngdoc

```diff
@@ -283,9 +283,12 @@ Here is an example of using the injector service:
 // Provide the wiring information in a module
 var myModule = angular.module('myModule', []);
 ```
+Notice that `myModule` is independent as array is empty here. 
+Dependent is always need dependency object to be injected inside it.
```

Comment:
I don't understand this sentence. Remember that the previous code snippet is defining a module so it is not helpful to refer to "objects" here.